target/dita/regions/Iceland/iceland_complex/iceland_complex.dita

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" ?>
+<!DOCTYPE classlist
+  SYSTEM '../../../dtd/classlist.dtd'>
+<classlist>
+	<title>Iceland - Complex</title>
+	<flag>
+		<fig>
+			<image alt="flag" href="images/iceland_flag.png"/>
```

Comment:
this line needs to be `../content/images/Flag.png`

Reply:
ok, but when i check the folder `britain/content/images` i see a `Flag.jpg` file there but not `Flag.png` and when i check the `iceland/content/images` folder i do have a `Flag.png` file

Reply:
I see...  Investigating.